flink-doris-connector/src/test/resources/autoci/e2e/mysql2doris/initialize/mysql_tbl1.sql

```diff
@@ -0,0 +1,8 @@
+DROP TABLE IF EXISTS test_e2e_mysql.tbl1;
+
+CREATE TABLE test_e2e_mysql.tbl1 (
+    `name` varchar(256) primary key,
+    `age` int
+);
+
+insert into test_e2e_mysql.tbl1 values ('doris_1',1);
```

Comment:
The table structures are the same, only the table names are different. Do we need to extract them?

Reply:
This is the next plan. These cases will be transformed. Each table will have a different structure and will be more complex. Now these cases are too simple.

Reply:
For functional testing, can this table structure be reused? And for all types of tests, a single table can be written. I feel that there are too many identical table structures here, which doesn't seem to make much sense?